config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
@matthewd see top from a 5.4 appliance running ruby 2.0 and postgresql 9.2, note the postgres processes at the bottom... stale connections probably won't be that big but they are definitely not free as in :beer: 

Does it make sense to spin up an appliance with these changes to compare before and after?

![screen shot 2015-07-10 at 10 37 14 am](https://cloud.githubusercontent.com/assets/19339/8620770/c4e4be5e-26ef-11e5-8672-00775c5309d8.png)